### ISSUE TITLE
AttributeError

### ISSUE THREAD

**Issue by volle1212** [dray-cyber/googlecontroller#8]
When I run this code:

`from googlecontroller import GoogleAssistant

host = "192.186.86.26"
home = GoogleAssistant(host=host)
home.say("test")`

I get this error:

`Some sort of error occured most likely a bad connection or ip adress
Traceback (most recent call last):
  File "E:\Programmering\python\Pycharm\Projects\mellocounter\main.py", line 5, in <module>
    home.say("test")
  File "C:\Users\folck\anaconda3\envs\interpeter change\lib\site-packages\googlecontroller\__init__.py", line 54, in say
    self.play(url, ignore)
  File "C:\Users\folck\anaconda3\envs\interpeter change\lib\site-packages\googlecontroller\__init__.py", line 30, in play
    if self.cc.media_controller.status.player_state != "PLAYING" or ignore == True:
AttributeError: 'GoogleAssistant' object has no attribute 'cc'`

I may have some internet thing that is not working but why the error?

**Comment by dray-cyber** [comment]
TLDR:
Pychromecast(cc) did not start(```Some sort of error occurred most likely a bad connection or ip address Traceback (most recent call last): File```) and therefore GoogleAssistant did not have the attribute as it never created.

**Comment by dray-cyber** [comment]
You can see it relates to attribute cc via this line:
```self.cc = pychromecast.Chromecast(host)``` in __init__.py

**Comment by dray-cyber** [comment]
Likely a wifi issue try unplugged/restarting your google home and checking your wifi. I will reopen this issue if you leave a comment here stating the problem still happens.
